scene/2d/tile_map.cpp

```diff
@@ -893,6 +893,34 @@ void TileMap::set_cell(int p_x, int p_y, int p_tile, bool p_flip_x, bool p_flip_
 	used_size_cache_dirty = true;
 }
 
+void TileMap::set_cell_region(const Vector2 &p_start, const Vector2 &p_end, int p_tile, bool p_flip_x, bool p_flip_y, bool p_transpose, Vector2 p_autotile_coord) {
```

Comment:
Instead of P_start and p_end, you should use a single Rect2i. See https://docs.godotengine.org/en/latest/classes/class_rect2i.html

Reply:
And to simplify the implementation, you could simply use p_rect = p_rect.abs(), as it automatically does what your are doing in the lines after.

Reply:
Ok, thank you for the advice.